src/query/functions-v2/src/scalars/comparison.rs

```diff
@@ -270,7 +291,13 @@ fn register_like(registry: &mut FunctionRegistry) {
 }
 
 fn vectorize_regexp(
-    func: impl Fn(&[u8], &[u8], &mut HashMap<Vec<u8>, Regex>, FunctionContext) -> Result<bool, String>
+    func: impl Fn(
+        &[u8],
+        &[u8],
+        &mut HashMap<Vec<u8>, Regex>,
+        &mut HashMap<Vec<u8>, String>,
+        FunctionContext,
```

Comment:
In general, the `FunctionContext` parameter is put first. We can. refine them all later in another PR.

Reply:
ok